docs/cooperation/cooperation-schema.yaml

```diff
@@ -151,14 +107,7 @@ definitions:
         minLength: 30
         maxLength: 1000
       proficiencyLevel:
-        type: string
-        enum:
-          - Beginner
-          - Intermediate
-          - Advanced
-          - Test Preparation
-          - Professional
-          - Specialized
+        $ref: '#/definitions/offerProficiencyLevelEnum'
```

Comment:
that enum is the same for request and cooperation, so you can change the name to `proficiencyLevelEnum`